packages/@featherds/snackbar/src/components/FeatherSnackbar.vue

```diff
@@ -84,14 +85,14 @@ export default defineComponent({
         remaining = duration;
       const pause = () => {
         if (!id) return;
-        window.clearTimeout(id);
+        clearTimeout(id);
```

Comment:
should this use our _clearTimeout?

Reply:
To resolve the immediate issue, no, because the return value isn't being used. May replace all of these later though.